src/utils/configurationVault/getConfiguration.js

```diff
@@ -9,6 +9,7 @@ const DEFAULT_CONFIGURATION = {
   [CONFIG.AUTO_ADD]: false,
   [CONFIG.EMOJI_FORMAT]: EMOJI_COMMIT_FORMATS.CODE,
   [CONFIG.SCOPE_PROMPT]: false,
+  [CONFIG.MESSAGE_PROMPT]: true,
```

Comment:
If someone already has a configuration initialised (locally) they'll have to run the `gitmoji -g` again to set this new key.

By default after adding this the message prompt will disappear for everyone 😓 

I'm wondering if we should somehow take this into account here:

https://github.com/carloscuesta/gitmoji-cli/blob/276713b7f6a30b7fd3722aa481543644099d5b6d/src/utils/configurationVault/getConfiguration.js#L55-L62

So we can provide the fallback for values that are newly introduced

Reply:
Easiest solution would be to utilise the `default` schema property from `conf` but that won't apply to users that use a local config in their projects. I guess we could do both, provide the defaults in schema from the `DEFAULT_CONFIG` as well as falling back to `DEFAULT_CONFIG[key]` in case `configuration[key]` doesn't return anything.

Reply:
Check out e06760deb4a0cf2911b515bf4503456d622d1aeb

Reply:
Yep! That makes a lot of sense 🙏🏼 🚀